libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SFLoginViewController.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I noticed that too, I am going to move them to physical folders on the file system. Thanks for pointing it out.